frontend/src/metabase/visualizations/components/ChartClickActions.css

```
@@ -0,0 +1,34 @@
+.token {
+  padding: 5px 14px 5px 14px;
```

No need to duplicate here if the X and Y values are going to be the same on each axis.
```suggestion
  padding: 5px 14px;
```